samples/hello_events.c

```diff
@@ -115,12 +131,120 @@ static fpga_result inject_ras_fatal_error(fpga_token token, uint8_t err)
 	return result;
 }
 
+/*
+ * Parse command line arguments
+ */
+#define GETOPT_STRING "B:"
+fpga_result parse_args(int argc, char *argv[])
+{
+	struct option longopts[] = {
+		{"bus", required_argument, NULL, 'B'}
```

Comment:
longopts needs to be NULL-terminated.  ie
{ "bus", required_argument, NULL, 'B' },
{ NULL, 0, NULL, 0 }